pkg/constants.go

```diff
@@ -30,5 +30,5 @@ const (
 	ConnectionTemplatesAPIPath = "v1/connection/connection-templates"
 	ConsumersGroupPath         = "api/consumers"
 	ElasticsearchIndexesPath   = "/api/elastic/indexes"
-	AlertsSettingsPath         = "api/v1/alerts/settings"
+	AlertsSettingsPath         = "api/v1/alert/settings"
```

Comment:
is this backward compatible?

Reply:
This is the new v1 endpoint that was changed just before the release of 3.1